__tests__/infrastructure/NodeService.spec.js

```diff
@@ -0,0 +1,229 @@
+import { NodeService } from '../../src/infrastructure';
+import http from '../../src/infrastructure/http';
+import TestHelper from '../TestHelper';
+
+describe('Node Service', () => {
+	// Arrange:
+	const {
+		generateNodePeerStatus,
+		generateNodeApiStatus,
+		nodeCommonField
+	} = TestHelper;
+
+	const statisticServiceNodeResponse = [
+		{
+			roles: 1,
+			peerStatus: generateNodePeerStatus(true),
+			...nodeCommonField
+		},
+		{
+			roles: 2,
+			apiStatus: generateNodeApiStatus(false),
+			...nodeCommonField
+		},
+		{
+			roles: 3,
+			peerStatus: generateNodePeerStatus(true),
+			apiStatus: generateNodeApiStatus(false),
+			...nodeCommonField
+		},
```

Comment:
why does this node not show up in list since it is reachable via peer api?

Reply:
The reason is that the API node is down.

I did the logic check here

https://github.com/symbol/explorer/pull/1167/files#diff-80bd52740de6d63768e32a7ecd0f2980c8f7b135ff2731d2a314cf5b8252c9b4R102-R103

Reply:
yes, but if peer is operating it will still be participating in underlying network even if REST is down. so, i'm not sure hiding it is the correct behavior.

Reply:
yeah, it makes sense.
I will update the logic.